Detectors/MUON/MCH/Simulation/src/Materials.cxx

```diff
@@ -189,12 +190,17 @@ void createMaterials()
   float wPlastic[nPlastic] = { 1, 1 };
   float dPlastic = 1.107;
   mgr.Mixture(kModuleName, ++imat, "Plastic", aPlastic, zPlastic, dPlastic, -nPlastic, wPlastic);
-  mgr.Medium(kModuleName, Medium::Plastic, "Plastic", imat, 0, fieldType, maxField, kMaxfd, kStemax, kDeemax, kEpsil,
+  mgr.Medium(kModuleName, Medium::Plastic, "Plastic", imat, kIsUnsens, fieldType, maxField, kMaxfd, kStemax, kDeemax, kEpsil,
              kStmin);
 
-  /// Epoxy (definition taken from AliMUONCommonGeometryBuilder)
-  mgr.Material(kModuleName, ++imat, "Epoxy", 12.24, 6., 1.85, 0., 0.);
-  mgr.Medium(kModuleName, Medium::Epoxy, "Epoxy", imat, 0, fieldType, maxField, kMaxfd, kStemax, kDeemax, kEpsil,
+  /// Epoxy : C18 H19 O3 (to be confirmed)
```

Comment:
Hi Florian,
Not a big fan of (to be confirmed, todo, etc...) in the code. You'd better put that one(s) in JIRA instead (or in addition to... then should remember to remove the comments in the code once this is settled)